ceph-client.rb

```diff
@@ -32,6 +32,11 @@ class CephClient < Formula
     sha256 "b8eac752c5e14d3eca0e6dd9199cd627518cb5ec06add0de9d32baeee6fe645d"
   end
 
+  resource "wcwidth" do
+    url "https://files.pythonhosted.org/packages/89/38/459b727c381504f361832b9e5ace19966de1a235d73cdbdea91c771a1155/wcwidth-0.2.5.tar.gz"
+    sha256 "c4d647b99872929fdb7bdcaa4fbe7f01413ed3d98077df798530e5b04f116c83"
+  end
```

Comment:
hi @thmour , do you have the python backtrace caused by missing `wcwidth`? if yes, could you paste here? as i am not able to find the direct dependency on this python package by grep'ing it in the quincy tree:
```
$ git grep wcwidth
src/pybind/mgr/dashboard/frontend/package-lock.json:        "wcwidth": "^1.0.1"
src/pybind/mgr/dashboard/frontend/package-lock.json:    "wcwidth": {
src/pybind/mgr/dashboard/frontend/package-lock.json:      "resolved": "https://registry.npmjs.org/wcwidth/-/wcwidth-1.0.1.tgz",
```

Reply:
```
thmour@Theofilos-MacBook-Pro homebrew-ceph-client % ceph -s
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/ceph-client/quincy-17.0.0-4483-g0def4cf21d8/libexec/bin/ceph", line 153, in <module>
    from ceph_daemon import admin_socket, DaemonWatcher, Termsize
  File "/opt/homebrew/lib/python3.9/site-packages/ceph_daemon.py", line 24, in <module>
    from prettytable import PrettyTable, HEADER
  File "/opt/homebrew/Cellar/ceph-client/quincy-17.0.0-4483-g0def4cf21d8/libexec/vendor/lib/python3.9/site-packages/prettytable/__init__.py", line 1, in <module>
    from .prettytable import (
  File "/opt/homebrew/Cellar/ceph-client/quincy-17.0.0-4483-g0def4cf21d8/libexec/vendor/lib/python3.9/site-packages/prettytable/prettytable.py", line 45, in <module>
    import wcwidth
ModuleNotFoundError: No module named 'wcwidth'
```

Reply:
thanks!